src/service/viewport-impl.js

```diff
@@ -645,9 +645,6 @@ export class ViewportBindingNatural_ {
     if (doc./*OK*/scrollingElement) {
       return doc./*OK*/scrollingElement;
     }
-    if (doc.body) {
```

Comment:
Are you sure this is correct? There are still lots of chromes out there with the old API...